src/components/People.tsx

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable prefer-template */
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
try to disable eslint as little as possible, it's actually pretty usefel 😄

Reply:
+1